image/ansible-devops/env.sh

```diff
@@ -41,7 +41,6 @@ export CPD_STORAGE_CLASS=$(cat /workspace/settings/CPD_STORAGE_CLASS)
 # -----------------------------------------------------------------------------
 # cp4d/install-db2=api.yml
 # -----------------------------------------------------------------------------
-export CPD_DB2WH_INSTANCE_NAME=$(cat /workspace/settings/CPD_DB2WH_INSTANCE_NAME)
```

Comment:
Is removal of this environment variable definitely safe? 

Although it is assigned a default in the `mas/hack-manage-db2.yml` playbook:
```
    db2wh_instance_id: "{{ lookup('env', 'CPD_DB2WH_INSTANCE_NAME') | default('db2wh-db01', true) }}"
```
This is not the case in the `cp4d/install-db2-api.yml` playbook:
```
cp4d_db2wh_instance_name: "{{ lookup('env', 'CPD_DB2WH_INSTANCE_NAME') }}" # e.g. db2w-iot or db2w-manage
```

Not saying this is incorrect - I don't fully understand what's going here - just wanted you to double check.

Reply:
This variable is in the scope of the ClusterTask, so we set that when using the ClusterTask in the pipeline. If we keep the car in .env, considering that .env is sourced inside the ClusterTask right before the playbook call, it will replace the parameter set to the task (clearing it out) and playbook execution will fail. Apologize for not adding the appropriate links here but will do as soon as I get to my desk!